Documentation.md

```diff
@@ -197,6 +197,17 @@ this:
 
 ## `connect.core`
 
+### `connect.core.initialized`
+```js
+if (connect.core.initialized) { /* ... */ }
```

Comment:
Customers are using this internal field without knowing that it's not what they think it is, and that it should not be used. I am fine with removing it from TypeScript to discourage its usage though.

That being said, I still see value in putting a warning against its usage, so customers don't walk into a trap if they use it. How about removing the documentation but putting the following at the end of the `## connect.core` section:

> **Note about `connect.core.initialized`:** This field is internally used by `amazon-connect-streams` and it's not part of the public API. Customers are discouraged to use it to determine whether the CCP has been initialized since it represents a different internal state. As an alternative, consider using `connect.agent()`.